maitake/src/scheduler.rs

```diff
@@ -888,7 +888,20 @@ impl Core {
             has_remaining: false,
         };
 
-        for task in self.run_queue.consume().take(n) {
+        while tick.polled < n {
+            let task = match self.run_queue.try_dequeue() {
+                Ok(task) => task,
+                // If inconsistent, just try again.
```

Comment:
nit, take it or leave it: maybe worth mentioning that the reason the outer loop is not a `for` loop is so that we don't increment `tick.polled` in this case? or maybe that's obvious, i dunno...